ee/billing/billing_manager.py

```diff
@@ -138,7 +138,7 @@ def update_billing(self, organization: Organization, data: dict[str, Any]) -> No
 
     def update_billing_organization_users(self, organization: Organization) -> None:
         try:
-            distinct_ids = list(organization.members.values_list("distinct_id", flat=True))
+            distinct_ids = list(organization.members.values_list("distinct_id", flat=True))  # type: ignore
```

Comment:
I don't understand why these started throwing errors in this PR. I didn't make any changes related.

Reply:
Needed to ignore and update baseline